### ISSUE TITLE
Install SBT with an Authenticated Proxy Repository

### ISSUE THREAD

**Issue by nunoneto** [sbt/sbt#2026]
Hi,

I'me new to sbt and am trying to install sbt from inside a corporate network, using a coporate instance of OSS Nexus. However, from all the documentation I've read, there doesn't seem to be any global authentication support for proxy repositories. In summary, I can't even install activator, so no chance to configure authentication at a project level..
Is it possible to add this feature in a future release?

Thanks

**Comment by jsuereth** [comment]
Yes.  This is actually an http://github.com/sbt/launcher issue.  I think there may have been activity here, but I'll need to check the git logs.
